BackofficeBundle/Resources/public/coffee/groupTree/NodeTreeView.coffee

```diff
@@ -16,29 +17,33 @@ NodeTreeView = OrchestraView.extend(
     @options.domContainer = @$el
 
   render: ->
-    @options.domContainer.html @renderTemplate('OpenOrchestraBackofficeBundle:BackOffice:Underscore/groupTree/nodeTree')
-    @options.domContainer.append @renderTemplate('OpenOrchestraBackofficeBundle:BackOffice:Underscore/backToList',
-      listUrl : @options.listUrl
-    )
-    currentView = @
-    $.ajax
-      url: currentView.options.html.links._self_node_tree
-      method: "GET"
-      success: (response) ->
-        currentView.options.nodes = response
+    if typeof @options.response.site != 'undefined'
+        @options.domContainer.html @renderTemplate('OpenOrchestraBackofficeBundle:BackOffice:Underscore/groupTree/nodeTree')
+        currentView = @
         $.ajax
-          url: currentView.options.html.links._role_list_node
+          url: currentView.options.response.links._self_node_tree
```

Comment:
if there isn't this param, there is an error of development so the script should not wor.